lessons/1e_camera.md

```diff
@@ -0,0 +1,70 @@
+
+## Setting up a Camera
+
+To use the `Render` mode of rendering, we have to add a camera. This can be done easiest in the `Layout` tab. 
+Here press `Add` or `Shift+A` and add a camera.
+
+You can look through the current `Active Camera` by clicking the camera icon in the `3D viewport` panel (`Active Camera` is settable in the `Outliner` panel by clicking the green camera, if only one camera is present, this is the active camera)
+
+ <img src="../figures/view through camera.png" width="200"/>
+
+**For an easy top view:**
+In `Object Properties` set the camera `Location` to 0 in `X`/`Y` and above the volume in `Z`, and `Rotation` to 0 for all. 
```

Comment:
I find it easier to click on the `Z` axis, adjust zoom and do the align camera to viewport...